js/Landing.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+   color: white;
+`;
+
+const LandingNavbar = styled.div`
+  h2{
+    float:left;
+    margin:2px;
+    font-weight: normal;
+  }
+  a{
+    float:right;
+    margin:5px;
+    font-weight: bolder;
+  }
+  font-family: Roboto
+
+`;
+
+const LandingHeader = styled.div`
+text-align: center;
+word-wrap: break-word;
+padding-top: 50px;
+padding-bottom: 10vh;
+font-family: 'Questrial';
+font-size: 150%;
+button{
+  font-family: 'Questrial';
+  font-weight: bold;
+  font-size: 100%;
+  background-color: #F7b733;
+  border-radius: 100px;
+  border: none;
+  color: white;
+  padding:15px 40px;
+}
+@media (max-width: 700px){
+  h1{
+    font-size: 100%;
+  }
+}
+`
+const LandingGraph = styled.div`
+
+  background-color: #f0ebe1;
+  text-align: center;
+
+h2{
+
+}
+select{
+  font-family: 'Questrial';
+  font-weight:bold;
+  font-size: 120%;
+  background-color: #F7b733;
+  border-radius: 100px;
+  border: none;
+  color: white;
+  padding:15px 40px;
+}
+`;
+const LandingSafety = styled.div`
+  padding: 10vh;
```

Comment:
non-urgent, but make sure you test this on a few devices to ensure that 10% of the vertical height of the page is the right amount of padding in all circumstances - or add media queries!